db/migrate/20190926234420_create_charts.rb

```diff
@@ -0,0 +1,8 @@
+class CreateCharts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :questions do |t|
+      t.numeric :chart_id
```

Comment:
We don't know if the table will be called `Chart` yet, so let's leave this out for now.

Reply:
I changed it to questions. 

I thought it would be good to have this create a table for now as an example for how the DB gets initialized. There's seed data and a sample endpoint too. I'm sure it can be changed easily in the future by whoever is working on the database schema. Lmn what you think.